corehq/apps/export/views.py

```diff
@@ -1673,6 +1675,16 @@ def _check_deid_permissions(self, export_instances):
                         "De-Identified export.")
                     )
 
+    def _check_export_size(self, export_instances, filters):
+        count = 0
+        for instance in export_instances:
+            count += get_export_size(instance, filters)
+        if count > MAX_EXPORTABLE_ROWS:
```

Comment:
hmm i see you use hte total count in the error message. might be worth leaving as is then